build.xml

```diff
@@ -221,7 +214,15 @@
       <arg value="--always"/>
     </exec>
     <!-- Set version if git describe failed. -->
-    <property name="adaptorlib.version" value="unknown"/>
+    <property name="adaptorlib.version" value="unknown-${DSTAMP}"/>
```

Comment:
I should say, "unknown" is pretty rare, because you basically have to not be using the repository with git, but changing the version to include the timestamp and the suffix to include a warning that we don't have a real version both seemed like mild improvements to a corner case.